### DISCUSSION TITLE
Does Hyperloop support Swift Package Manager?

### DISCUSSION THREAD

**Discussion by dazza5000** [appcelerator/titanium_mobile#12699] [Q&A]
I know it supports cocoapods and custom frameworks.

Does Hyperloop support Swift Package Manager?

**Comment by hbugdoll** [comment]
The support of [SPM](https://docs.swift.org/swiftpm/documentation/packagemanagerdocs/) in the SDK (not only for Hyperloop) is currently in progress: https://github.com/tidev/titanium-sdk/pull/14327

**Comment by hbugdoll** [comment]
SPM is now supported with Ti SDK 13.1.0 🚀
